src/modules/settings/components/Table.jsx

```diff
@@ -97,6 +97,17 @@ function EditCell({rowData, dataKey, onChange, onMouseOver, onMouseLeave, onClic
     case Status.EDIT: {
       const wrapperRef = useRef(null);
       useOutsideAlerter(wrapperRef, onClick, rowData);
+
+      useEffect(() => {
```

Comment:
im surprised we don't have linting errors, but you cannot reliably use hooks unless they are the first thing being executed in a function. https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
this applies also to the ones above.